src/transformers/models/phi/modeling_phi.py

```diff
@@ -192,7 +212,6 @@ def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
         return hidden_states
 
 
-# Copied from transformers.models.persimmon.modeling_persimmon.PersimmonAttention with Persimmon->Phi,persimmon->phi
```

Comment:
that's not something we want to remove, we should also have flash attention support in persimmon should be the same as this

Reply:
Actually, there is another [PR](https://github.com/huggingface/transformers/pull/27052) which is adding the FA support for Persimmon. 

Should I add the `self.causal=True` in the `PersimmonAttention` so that we can keep this `# Copied from` statement?

Reply:
Yeah! 🤗

Reply:
Done @ArthurZucker.